pandas/tests/extension/test_arrow.py

```diff
@@ -1125,6 +1125,27 @@ def test_comp_masked_numpy(self, masked_dtype, comparison_op):
         expected = pd.Series(exp, dtype=ArrowDtype(pa.bool_()))
         tm.assert_series_equal(result, expected)
 
+    def test_groupby_agg_extension(self, data_for_grouping):
```

Comment:
I think this test should behave the same as the one in the base class. If that's the case, this can be removed. Can you confirm?

Reply:
resolved